docs/compliance/cis/docker_ee.md

```diff
@@ -0,0 +1,35 @@
+---
+description: CIS Docker EE Benchmark information
+keywords: standards, compliance, security, benchmark
+title: CIS Docker EE benchmark
+---
+
+In January 2018, in partnership with the Center for Internet Security (CIS) and
+Microsoft, we began to collaborate on an updated version of the CIS Docker
+Benchmark to incorporate Docker Enterprise Edition security recommendations.
+This includes Docker EE Engine, Docker Trusted Registry (DTR), and Universal
+Control Plane (UCP). These updates will also include configuration items for
+Docker EE Engine on Windows Server, Windows Server host items, and Windows-based
+Docker image recommendations. We anticipate completion of the draft and final
+update of the Benchmark in the February timeframe.
+
+You can follow along and contribute to the development of this updated Benchmark
+by joining the [CIS Docker Benchmarks
+Community](https://workbench.cisecurity.org/communities/37) in CIS WorkBench.
+The initial announcement of these updates can also be found in the Community
+Discussions
+[here](https://workbench.cisecurity.org/community/37/discussions/3349).
+
+In the interim, you can reference the existing CIS Docker CE Benchmark
+recommendations per our guidance [here](/compliance/cis/docker_ce.md) and refer
+to our latest [Docker EE Security Reference
+Architecture](http://success.docker.com/article/Docker_Reference_Architecture-_Securing_Docker_EE_and_Security_Best_Practices).
+We will update this page accordingly once the new CIS Benchmark updates have
```

Comment:
This is not really important, but if you see instances like this where an unbalanced underscore makes the whole rest of your document look like italics, you can add a class to try to trick the Atom renderer, perhaps after a paragraph:

```none
{:class="_"}
```

Reply:
Does this work for VSCode as well? I'm currently using the [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) extension for VSCode which allows me wrap lines at 80 characters with a simple shortcut